modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -2,6 +2,9 @@
 
 You can use the Runtime Fabric command line tool, `rtfctl`, to install Anypoint Runtime Fabric on an Amazon Elastic Kubernetes Service (Amazon EKS), Amazon Elastic Kubernetes Service Anywhere (Amazon EKS-A), Azure Kubernetes Service (AKS) or Google Kubernetes Engine (GKE) installation that you manage. 
 
+[NOTE]
+Runtime Fabric v2.2.5 with multi-tenancy support requires `rtfctl` 0.4.x.
```

Comment:
We use `multiple instances` support instead of multi-tenancy in all our docs. e.g. https://docs.mulesoft.com/runtime-fabric/2.3/install-multiple-instances should we be consistent with that?

Reply:
Also, we should specify > 2.2.5 and > 0.4?

Reply:
And this merges only into `2.3`, should we also merge into `2.2` so users viewing under `2.2` can see it?

Reply:
+1 that will be more intuitive and consistent with our other documentation.